samples/sessions_list.py

```diff
@@ -66,22 +66,28 @@
         prompt='Enter password for host %s and user %s: ' %
                (args.host, args.user))
 
-si = connect.SmartConnect(host=args.host,
-                          user=args.user,
-                          pwd=password,
-                          port=int(args.port))
+si = SmartConnectNoSSL(
+        host=args.host,
+        user=args.user,
+        pwd=password,
+        port=int(args.port)
+    )
 
-print "logged in to %s" % args.host
+print("logged in to %s" % args.host)
```

Comment:
Hi @jm66 Thanks for making this change. Please can you make quick change, replace %s substitution with  string.format() method instead.

Reply:
Will do. Thanks @pgbidkar